src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/VetFieldEnum.java

```diff
@@ -148,8 +171,8 @@ public String getColumnValue(final VariantContext variant) {
 
     AS_RAW_ReadPosRankSum {  // TODO -- maybe rely on 1/1 for call_GT
         public String getColumnValue(final VariantContext variant) {
-            String out =  getAttribute(variant, GATKVCFConstants.AS_RAW_READ_POS_RANK_SUM_KEY, "");
-            if (out.contentEquals("||") || out.contentEquals("|||") ) {
+            String out =  getAttribute(variant, GATKVCFConstants.AS_RAW_READ_POS_RANK_SUM_KEY, null);
+            if (out == null || out.contentEquals("||") || out.contentEquals("|||") ) {
                 out = " "; // TODO is this better than null?
```

Comment:
I didn't notice before that we were setting this to a space (here and for MQRankSum). for PTG and PID we use the empty string if we don't have a value. can we change these to the empty string too?